pkg/sql/parser/sql.y

```diff
@@ -3114,6 +3124,88 @@ create_schedule_for_backup_stmt:
   }
  | CREATE SCHEDULE error  // SHOW HELP: CREATE SCHEDULE FOR BACKUP
 
+
+// %Help: ALTER BACKUP SCHEDULE - alter an existing backup schedule
+// %Category: CCL
+// %Text:
+// ALTER BACKUP SCHEDULE <id>
+// {
```

Comment:
nit: square brackets usually used for this?

Reply:
Oh, I like the ALTER TABLE formulation, thanks! Makes it clearer that at least one is required.